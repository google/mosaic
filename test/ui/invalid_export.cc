namespace rust_export
{
    struct FooBar
    {
    };

    int baz;
} // namespace rust_export
