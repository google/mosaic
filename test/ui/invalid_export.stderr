error: invalid rust_export item
   ┌─ $DIR/invalid_export.cc:11:5
   │
11 │ ╭     struct FooBar
12 │ │     {
13 │ │     };
   │ ╰─────^ only using declarations are allowed here

error: invalid rust_export item
   ┌─ $DIR/invalid_export.cc:15:5
   │
15 │     int baz;
   │     ^^^^^^^ only using declarations are allowed here
